gameengine.js

```diff
@@ -54,7 +54,7 @@ GameEngine.prototype.start = function start() {
   const that = this;
   (function gameLoop() {
     that.loop();
-    requestAnimFrame(gameLoop, that.ctx.canvas);
+    window.requestAnimFrame(gameLoop, that.ctx.canvas);
```

Comment:
On this line, eslint was complaining about requestAnimFrame not being a function so I just added the window variable to access its requestAnimFrame function